<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>270</width>
    <height>219</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>小小按键精灵</string>
  </property>
  <widget class="QWidget" name="layoutWidget_switch">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>60</y>
     <width>171</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="form_switch">
    <item row="0" column="0">
     <widget class="QLabel" name="label_switch">
      <property name="text">
       <string>切换：</string>
      </property>
      <property name="buddy">
       <cstring>keySequence_switch</cstring>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QKeySequenceEdit" name="keySequence_switch"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_key1">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>140</y>
     <width>171</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="form_key1">
    <item row="0" column="0">
     <widget class="QLabel" name="label_key1">
      <property name="text">
       <string>键一：</string>
      </property>
      <property name="buddy">
       <cstring>keySequence_key1</cstring>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QKeySequenceEdit" name="keySequence_key1"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_start">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>211</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="form_start">
    <item row="0" column="0">
     <widget class="QLabel" name="label__start">
      <property name="text">
       <string>  开始/停止：</string>
      </property>
      <property name="buddy">
       <cstring>keySequence_start</cstring>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QKeySequenceEdit" name="keySequence_start"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_key2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>180</y>
     <width>171</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="form_key2">
    <item row="0" column="0">
     <widget class="QLabel" name="label_key2">
      <property name="text">
       <string>键二：</string>
      </property>
      <property name="buddy">
       <cstring>keySequence_key2</cstring>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QKeySequenceEdit" name="keySequence_key2"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_interval">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>261</width>
     <height>26</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="hor_interval">
    <item>
     <widget class="QLabel" name="label_interval">
      <property name="text">
       <string>        间隔：</string>
      </property>
      <property name="buddy">
       <cstring>spinBox_interval</cstring>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="spinBox_interval"/>
    </item>
    <item>
     <widget class="QLabel" name="label_unit">
      <property name="text">
       <string>毫秒         </string>
      </property>
      <property name="buddy">
       <cstring>spinBox_interval</cstring>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>spinBox_interval</tabstop>
  <tabstop>keySequence_switch</tabstop>
  <tabstop>keySequence_start</tabstop>
  <tabstop>keySequence_key1</tabstop>
  <tabstop>keySequence_key2</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
